api/services/session_test.go

```diff
@@ -44,41 +32,56 @@ func TestListSessions(t *testing.T) {
 	cases := []struct {
 		name          string
 		pagination    paginator.Query
-		requiredMocks func()
+		requiredMocks func(query paginator.Query)
```

Comment:
Should we instantiate the `query` inside the `requiredMocks` as in other tests?

I think I'll enforce the test and make it broken if `query` changes in some way. Am I right?

Reply:
corrected in other cases